MarkupSafe vs Jinja dependency issues

I'm trying to run tests as part of #3866 .
 
 I'm not able to `make init`. I've had about 4 different errors.
 
 I'm trying to do it in Docker, hoping that doing it in a clean slate makes this reproducable. 

# Steps to reproduce

Add `/tests/Dockerfile`:

```
FROM python:3.7-buster

ADD . /moto/
ENV PYTHONUNBUFFERED 1
WORKDIR /moto/
RUN make init
RUN make test
```

From the top of the repo, run `docker build -f tests/Dockerfile .`

# Expected behavior

Build completes without error. That is, `make init` and `make test` ran without error, inside a clean Ubuntu Buster image.

# Actual behavior

```
Installed /usr/local/lib/python3.7/site-packages/boto3-1.17.57-py3.7.egg
error: MarkupSafe 1.1.1 is installed but MarkupSafe>=2.0.0rc2 is required by {'Jinja2'}
make: *** [Makefile:13: init] Error 1
The command '/bin/sh -c make init' returned a non-zero code: 2
```

# Solution

I'm about to push a PR. The dependency for Markup needs to change from < to != .

--- a/setup.py
+++ b/setup.py
@@ -40,7 +40,7 @@ install_requires = [
     "pytz",
     "python-dateutil<3.0.0,>=2.1",
     "responses>=0.9.0",
-    "MarkupSafe<2.0",  # This is a Jinja2 dependency, 2.0.0a1 currently seems broken
+    "MarkupSafe!=2.0.0a1",  # This is a Jinja2 dependency, 2.0.0a1 currently seems broken
 ]
 
 #

